services/QuillLMS/app/views/layouts/twenty_seventeen_home.html.erb

```diff
@@ -22,9 +22,9 @@
     <link rel="apple-touch-icon" href="<%= ENV['CDN_URL'].gsub(/[‘’]/, '') + "/images/logos/apple-touch-icon.png" %>">
     <link rel="canonical" href="https://www.quill.org<%= request.path %>">
 
-    <script defer src=<%= ENV['FONT_AWESOME_KIT_LINK'] %> crossorigin="anonymous" integrity="sha384-p7JrABPXxZLpj1XoHTzkPyVs8ekVssRFXc4B7XU6Z1c8XVDA7sVPem/lQ9UouxqE"></script>
+    <script defer src="<%= ENV['FONT_AWESOME_KIT_LINK'] %>" crossorigin="anonymous" integrity="sha384-p7JrABPXxZLpj1XoHTzkPyVs8ekVssRFXc4B7XU6Z1c8XVDA7sVPem/lQ9UouxqE"></script>
     <%= render partial: 'typekit' unless Rails.env.test? %>
-    <%= javascript_pack_tag((@js_file || 'home'), 'crossorigin': 'anonymous') %>
+    <%= javascript_pack_tag((@js_file || 'home'), defer: true, 'crossorigin': 'anonymous') %>
```

Comment:
This is the main change. I think this should be fine for the public pages that use this layout. For other React-heavy pages, it would cause a bit of a blank-page-then-pop-in effect, so I held off on updating the global header for now.